src/tool_listhelp.c

```diff
@@ -29,7 +29,7 @@
  * This source file is generated with the following command in an autotools
  * build:
  *
- * "make listhelp"
+ * "make -C docs/cmdline-opts/ listhelp"
```

Comment:
Also here.

Reply:
I can generate the source fine with `make listhelp` exactly as already documented.

Reply:
> I can generate the source fine with `make listhelp` exactly as already documented.
> 
> I invoke that command in the `src/` directory.

OK, that makes sense, I didn't know `src/` had it's own targets. Will remove this commit.